core/src/main/resources/mappings/scheduled-jobs.json

```diff
@@ -418,7 +418,7 @@
                         "monitor_id": {
                           "type": "keyword"
                         },
-                        "chained_findings": {
+                        "chained_monitor_findings": {
```

Comment:
do we need to fix this in common utils for the model also ?

Reply:
I see the PR https://github.com/opensearch-project/common-utils/pull/390/files
thank you